Griddly.NetCore.Razor/wwwroot/js/griddly.js

```diff
@@ -566,14 +566,14 @@
         this.setFilterValues(values, true, null, true);
     };
 
-    var hidePopover = function (selector, isBootstrap4) {
+    var hidePopover = function (selector, isBootstrap4or5) {
```

Comment:
Again, a isBootstrap4OrGreater would be more flexible

Reply:
I'm not sure why `hidePopover` needs an argument for this anyhow. It should really use feature detection instead I think. All the isBootstrap4 values passed around in griddly.js are just to enable calling hidePopover.  😢

Reply:
If we want it to support multiple frameworks, having an isBootstrap4or5 is probably not ideal. It would need to be a parameter which contains framework name and version.

Reply:
> I'm not sure why `hidePopover` needs an argument for this anyhow. It should really use feature detection instead I think. All the isBootstrap4 values passed around in griddly.js are just to enable calling hidePopover. 😢

I think the idea was not to do feature detection in every place because:
1) it's more verbose
2) it's less efficient

Makes more sense to do feature detection at the top level, and then expose the assessment to all methods.

Reply:
I suggest we cache the version instead in that case. Same as the compiled approach.